docs/guides/developing/usage_examples.rst

```diff
@@ -323,3 +323,56 @@ This example is available as a working app in the OCIO source: src/apps/ociodisp
    oglApp->setShader(shaderDesc);
 
    oglApp->redisplay();
+
+Python
+++++++
+
+.. code-block:: python
+
+   import PyOpenColorIO as OCIO
+
+   # Step 1: Get the config
+   config = OCIO.GetCurrentConfig()
+
+   # Step 2: Lookup the display ColorSpace
+   device = config.getDefaultDisplayDeviceName()
+   transformName = config.getDefaultDisplayTransformName(device)
+   displayColorSpace = config.getDisplayColorSpaceName(device, transformName)
+
+   # Step 3: Create a DisplayTransform, and set the input and display ColorSpaces
+   # (This example assumes the input is scene linear. Adapt as needed.)
+
+   transform = OCIO.DisplayTransform()
+   transform.setInputColorSpaceName(OCIO.Constants.ROLE_SCENE_LINEAR)
+   transform.setDisplayColorSpaceName(displayColorSpace)
+
+   # Step 4: Add custom transforms for a 'canonical' Display Pipeline
+
+   # Add an fstop exposure control (in SCENE_LINEAR)
+   gain = 2**exposure
+   slope3f = (gain, gain, gain)
+
+   cc = OCIO.CDLTransform()
+   cc.setSlope(slope3f)
+
+   transform.setLinearCC(cc)
+
+   # Add a Channel view 'swizzle'
+
+   channelHot = (1, 1, 1, 1) # show rgb
+   # channelHot = (1, 0, 0, 0) # show red
+   # channelHot = (0, 0, 0, 1) # show alpha
+   # channelHot = (1, 1, 1, 0) # show luma
+
+   lumacoef = config.getDefaultLumaCoefs()
+
+   m44, offset = OCIO.MatrixTransform.View(channelHot, lumacoef)
```

Comment:
In OCIO v2 this now returns a `MatrixTransform` instance, so no need for the additional `swizzle` object below.